pkgs/development/libraries/libepoxy/default.nix

```diff
@@ -40,18 +40,18 @@ stdenv.mkDerivation (rec {
 
   nativeBuildInputs = [ meson ninja pkg-config utilmacros python3 ];
 
-  buildInputs = [
+  buildInputs = if !stdenv.isDarwin then [
     libGL
     libX11
-  ] ++ lib.optionals stdenv.isDarwin [
+  ] else [
     Carbon
     OpenGL
   ];
 
   mesonFlags = [
     "-Dtests=${if doCheck then "true" else "false"}"
-  ]
-  ++ optional stdenv.isDarwin "-Dglx=yes";
+    "-Dglx=${if stdenv.isDarwin then "no" else "yes"}"
```

Comment:
Why do you want to set `-Dglx=yes` on non-Darwin? Wouldn't be `optional stdenv.isDarwin "-Dglx=no";` be sufficient?

Reply:
We are passing the `libGL`, so it would make sense to expect GLX extensions to be available.